src/commands/append.json

```diff
@@ -0,0 +1,51 @@
+{
+    "APPEND": {
+        "summary": "Append a value to a key",
+        "complexity": "O(1). The amortized time complexity is O(1) assuming the appended value is small and the already present value is of any size, since the dynamic string library used by Redis will double the free space available on every reallocation.",
+        "group": "string",
+        "since": "2.0.0",
+        "arity": 3,
+        "function": "appendCommand",
+        "command_flags": [
+            "write",
+            "denyoom",
+            "fast"
+        ],
+        "acl_categories": [
+            "write",
+            "string",
+            "fast"
+        ],
+        "key_specs": [
+            {
+                "flags": [
+                    "write"
+                ],
+                "begin_search": {
+                    "index": {
+                        "pos": 1
+                    }
+                },
```

Comment:
just to be on the safe side, did you validate that the COMMAND command output before and after this PR is the same (i.e. no copy paste errors or other bug in the pipeline of the processing of keyspecs and such)?

Reply:
will do

Reply:
ran it and indeed there was a bug in the generation script. fixed.